lib/circles/members/remove.js

```diff
@@ -0,0 +1,19 @@
+const _ = require("lodash");
+
+module.exports = function(req, res) {
+    // TODO assert that userId is member of circle
+    // TODO assert that authenticated user is member of or invited to circle
```

Comment:
authenticated user *has to be* member if they are removing someone else's user id

if they are in the invited-list, they are *only* allowed to remove themselves

I'd like to see this more clearly in the TODOs, just in case we forget them there for a while 😛